src/cryptography/hazmat/primitives/serialization.py

```diff
@@ -178,6 +186,10 @@ class PublicFormat(Enum):
     OpenSSH = "OpenSSH"
 
 
+class ParameterFormat(Enum):
+    DHParameter = "PKCS3"
```

Comment:
I think PKCS3 would be better if you look at compatibility to `PrivateFormat` and `PublicFormat`. Calling this DHParameter might also have problems if someone will want to add another DH parameters serialization. So I think PCKS3 is better.